backends/init.go

```diff
@@ -1,7 +1,7 @@
 package backends
 
-import "gopkg.in/gilmour-libs/gilmour-e-go.v5/backends/redis"
+import "gopkg.in/gilmour-libs/gilmour-e-go.v5/backends/sentinel"
 
-func MakeRedis(host, password string) *redis.Redis {
-	return redis.MakeRedis(host, password)
+func MakeRedis(host, password string) *sentinel.Redis {
```

Comment:
Having this method here makes no sense since there are now two drivers. We should have `backends.redis.New` and `backends.sentinel.New` to construct the backends. Also, these methods should return a `Backend`
It is also fine to have them both in a single package like `backends.redis.NewRedis` and `backends.redis.NewSentinel`. In fact, this might be better, since both need to use gore.